package.json

```diff
@@ -2,9 +2,25 @@
   "name": "cross-domain-safe-weakmap",
   "version": "1.0.10",
   "description": "Cross domain safe WeakMap.",
-  "main": "index.js",
+  "main": "dist/cross-domain-safe-weakmap.js",
+  "module": "dist/module",
   "scripts": {
-    "test": "gulp test"
+    "build": "npm run babel & npm run webpack && npm run flow:build",
```

Comment:
The publish.sh script does a bunch of stuff that it wouldn't be easy to compress into a single line. Seems like having `package.json` point to shell scripts should be a pretty standard thing, right? Writing a lot of code directly in `package.json` feels like a bit of a mess.

> Also I don't think a test before building is a good idea... What if you just want to know if it builds correctly without going through a million tests?

Then we can just run `run run webpack` right? Typically a build job doesn't just combine the files, it runs tests and lint and everything else that's needed too. I think that's a pretty well established convention.